modules/ROOT/Edits/endpoint-https-send.adoc

```diff
@@ -0,0 +1,98 @@
+= HTTP and HTTPS Send Endpoint Settings
+
+HTTP and HTTPS send endpoint settings configure where to send messages in the following contexts:
+
+* Send to partners
++
+Target endpoint in an outbound message flow to which translated B2B/EDI messages are sent to the partners either directly or via a 3rd party connection
++
+* Target at host
++
+Target endpoint in an inbound message flow to which translated application JSON and XML messages are sent to the backend systems
+
+A host, partner, or 3rd party connection can own HTTP and HTTPS send endpoints.
+
+Create HTTP and HTTPS send endpoints by configuring these settings on the host, partner, or 3rd party connection page:
+
+[%header%autowidth.spread]
+|===
+|Name |Description |Required
+| Owner
+| Partner, 3rd party connection, or host who owns the endpoint
+| Yes
+
+|Protocol
+| `HTTP` or `HTTPS`. The default is *HTTPS*.
+|Yes
+
+|Endpoint Name
+|Automatically-generated name that identifiers the endpoint, which you can modify manually later on
+|Yes
+
+|Description
+|User-supplied description of the endpoint's purpose and use
+|No
+
+|URL
+|Target HTTP or HTTPs service to which to send translated EDI, JSON, or XML messages. The URL must be accessible from the runtime environment that processes your B2B integrations.
+Enter a well-formed URL (including the path) that specifies where to send the request, for example:
+`+https://www.mulesoft.com/sendRequest?name=something+`
+|Yes
+
+|Connection Idle Timeout (milliseconds)
+|Connection idle timeout period in milliseconds. The default is `30000`.
+|Yes
+
+|Response Timeout (milliseconds)
+|Maximum time, in milliseconds, for which the request element blocks the execution of the flow that is waiting for the HTTP response
+|No
+|===
+
+== Authentication Settings
+
+Depending on how you set up your authentication policy, refer to the appropriate authentication type and descriptions here.
```

Comment:
This sounds a little awkward to me. What is the audience action here: just referring to this table but not actually setting something, as in peer subtopics?